### ISSUE TITLE
Update button in Package Manager Settings window should be focused when selecting by pressing Alt+U

### ISSUE THREAD

**Issue by v-luzh** [NuGet/Home#9537]
## Details about Problem
NuGet version: NuGet Client Dev\5.7.0.6598
VS Version: Master\30029.156
OS: Windows-10-Enterprise-RS5
## Detailed repro steps 
1. Launch VS and click menu "Tools" -> "Nuget Package Manager" -> "Package Manger Settings".
2. Select "Package Sources" in the left pane.
3. Press Alt+U. 
## Expected
Update button should be focused when selecting by pressing Alt+U.
## Actual
Update button in Package Manager Settings window is not focused when selecting by pressing Alt+U as attached screenshot. Users don’t know if it is selected or not.
## Notes
1. Move from internal bug 651997.
2. It's not a regression bug because it did repro on D15.8 27918.00 + NuGet Client Release-4.8.0-RTM 4.8.0.5348.

**Comment by v-luzh** [comment]
It still reproes on VS Main\35222.175 with implicit NuGet Client Dev\6.12.0.82.

**Comment by v-luzh** [comment]
Close this bug as Won't Fix since the feature changed in Unified Settings Options page.
